src/middlewared/middlewared/plugins/acme_protocol_/issue_cert.py

```diff
@@ -100,15 +100,15 @@ def issue_certificate(self, job, progress, data, csr_data):
                 except errors.ValidationError as e:
                     msg = ''
                     for authzr in e.failed_authzrs:
-                       msg += '\n Authorization for identifier %s failed.' % (
+                       msg += f'\n Authorization for identifier {%s} failed.' % (
                          authzr.body.identifier)
-                       msg += '\n Here are the challenges that were not fulfilled:'
+                       msg += f'\n Here are the challenges that were not fulfilled:'
                        for challenge in authzr.body.challenges:
                          msg += \
-                               '\n Challenge Type: %s' \
-                               '\n Error information: ' \
-                               '\n Type: %s' \
-                               '\n Details: %s \n\n' % (
+                               f'\n Challenge Type: {%s}' \
```

Comment:
@Ornias1993 it would be `f'\n Challenge Type: {challenge.chall.typ}' \`.
So you would not have to do `% ()` later